BeagleBone_black/src/navigation/navigation.cpp

```diff
@@ -94,6 +133,20 @@ void Navigation::update(ImuArray imus, ProximityArray proxis, DataPoint<uint32_t
   stripeCounterUpdate(stripe_count.value);
 }
 
+void Navigation::calibrationUpdate(ImuArray imus)
+{
+  for (unsigned int i = 0; i < data::Sensors::kNumImus; ++i) {
+    g_ += imus[i].acc.value;
```

Comment:
depending on the number of samples in the calibration phase, have you considered an online mean update strategy to have better precision? For integer types, I would be worried about overflow, but since you are using float, you might only lose resolution with high values.